test/unit/gateways/clearhaus_test.rb

```diff
@@ -389,20 +389,20 @@ def failed_authorize_response
 
   def successful_capture_response
     {
-        'id' => 'd8e92a70-3030-4d4d-8ad2-684b230c1bed',
-        'status' => {
-            'code' => 20000
+      'id' => 'd8e92a70-3030-4d4d-8ad2-684b230c1bed',
+      'status' => {
+        'code' => 20000
+      },
+      'processed_at' => '2014-07-09T11:47:28+00:00',
+      'amount' => 1000,
+      '_links' => {
+        'authorization' => {
+          'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda'
         },
-        'processed_at' => '2014-07-09T11:47:28+00:00',
-        'amount' => 1000,
-        '_links' => {
-            'authorization' => {
-                'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda'
-            },
-            'refunds' => {
-                'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda/refunds'
-            }
-        }
+          'refunds' => {
+            'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda/refunds'
+          }
```

Comment:
The spacing between this line and the next makes it look like there's a missing `}`. Curious why that is?

Reply:
Thanks for catching this issue! It turns out that all three of these lines needed to be moved to the left by one tab.

Reply:
I had to run JoeboCop to find it